### ISSUE TITLE
Runtime error when using NPM package and trying to include a custom XAML control

### ISSUE THREAD

**Issue by MaxBen93** [microsoft/react-native-xaml#132]
When trying to follow the steps in the [TechnicalGuide](https://github.com/microsoft/react-native-xaml/blob/main/TechnicalGuide.md) in order to be able to use a custom XAML control from a WinRT component DLL I'm producing, everything appears to work fine in terms of compilation but at runtime I am presented with the following error:

```ps1
Invariant Violation: View config getter callback for component `XamlControl` must be a function (received `undefined`).

This error is located at:
in XamlControl
in Unknown (at AppAccount.js:92)
in RCTView (at View.windows.js:52)
...
```

The same error appears regardless of whether I attempt to reference the custom XAML control in my JSX or if I try to use a more standard `Windows.UI.Xaml` control like `RadioButton`.

The most notable difference I can think of here is that instead of cloning this repo and running through the steps, I am consuming the NPM package.

If I revert back to consuming the NPM package with no modifications made (that is, not updating the codegen to include a newer WinUI version and/or my custom component's WinMD) then referencing a control like `RadioButton` works fine again.

**Comment by asklar** [comment]
I was able to repro this in a self contained app too. Not sure what is happening yet.
Repro project here: https://github.com/asklar/repro_viewconfig

**Comment by asklar** [comment]
This should be fixed by 0.0.45 now so closing the issue : ) (thanks @acoates-ms for the offline help!)
